sdk/cosmos/azure-cosmos/azure/cosmos/aio/container.py

```diff
@@ -198,16 +199,20 @@ async def create_item(
 
     @distributed_trace_async
     async def read_item(
-        self,
-        item,  # type: Union[str, Dict[str, Any]]
-        partition_key,  # type: Any
-        **kwargs  # type: Any
+            self,
+            item,  # type: Union[str, Dict[str, Any]]
+            partition_key,  # type: Any
+            max_integrated_cache_staleness_in_ms=None,  # type: Optional[int],
+            **kwargs  # type: Any
     ):
         # type: (...) -> Dict[str, Any]
         """Get the item identified by `item`.
 
         :param item: The ID (name) or dict representing item to retrieve.
         :param partition_key: Partition key for the item to retrieve.
+        **Provisional** parameter max_integrated_cache_staleness_in_ms
+        :param max_integrated_cache_staleness_in_ms: The max cache staleness for the integrated cache in milliseconds.
```

Comment:
@simorenoh  - please update the documentation similar to below lines-

```
/**
     * Gets the staleness value associated with the request in the Azure CosmosDB service. For requests where the {@link
     * com.azure.cosmos.ConsistencyLevel} is {@link com.azure.cosmos.ConsistencyLevel#EVENTUAL} or {@link com.azure.cosmos.ConsistencyLevel#SESSION}, responses from the
     * integrated cache are guaranteed to be no staler than value indicated by this maxIntegratedCacheStaleness.
     *
     * <p>Default value is null</p>
     *
     * <p>Cache Staleness is supported in milliseconds granularity. Anything smaller than milliseconds will be ignored.</p>
     *
     * @return Duration of maxIntegratedCacheStaleness
     */
```

Reply:
I did see that Java had this whole thing, but I'll have to talk to Anna on this amount of comments for a parameter since in Java the object has it's own methods as opposed to Python, and it might end up cluttering the method description.